TestMetricsHealth is failing

I see the newly added testacse "TestMetricsHealth" is failing 
```
root@ubuntu18:/go/src/go.etcd.io/etcd# PASSES='integration_e2e' ./test -run='TestMetricsHealth'
Running with TEST_CPUS: 1,2,4
Starting 'integration_e2e' pass at Thu Oct 11 05:03:13 UTC 2018
Running integration and e2e tests...
testing: warning: no tests to run
PASS
ok  	go.etcd.io/etcd/tests/e2e	0.027s [no tests to run]
=== RUN   TestMetricsHealth
--- PASS: TestMetricsHealth (0.26s)
=== RUN   TestMetricsHealth
--- FAIL: TestMetricsHealth (2.11s)
    metrics_test.go:196: expected '1' from /health, got "2"
=== RUN   TestMetricsHealth
--- FAIL: TestMetricsHealth (0.17s)
    metrics_test.go:196: expected '1' from /health, got "3"
FAIL
FAIL	go.etcd.io/etcd/integration	2.559s
root@ubuntu18:/go/src/go.etcd.io/etcd# 
```
GO Version:

```
root@ubuntu18:/go/src/go.etcd.io/etcd# go version
go version go1.11.1 linux/amd64
root@ubuntu18:/go/src/go.etcd.io/etcd#

```

Platform Info:

```
root@ubuntu18:/go/src/go.etcd.io/etcd# uname -a
Linux ubuntu18.04-icp-node-4 4.15.0-36-generic #39-Ubuntu SMP Mon Sep 24 16:19:09 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
root@ubuntu18:/go/src/go.etcd.io/etcd#
root@ubuntu18:/go/src/go.etcd.io/etcd# cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.1 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
root@ubuntu18:/go/src/go.etcd.io/etcd#
```

I'm seeing this issue in ppc64le as well

--- a/integration/metrics_test.go
+++ b/integration/metrics_test.go
@@ -188,11 +188,11 @@ func TestMetricsHealth(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	hv, err := clus.Members[0].Metric("etcd_server_health_success")
+	hv, err := clus.Members[0].Metric("etcd_server_health_failure")
 	if err != nil {
 		t.Fatal(err)
 	}
-	if hv != "1" {
-		t.Fatalf("expected '1' from /health, got %q", hv)
+	if hv != "0" {
+		t.Fatalf("expected '0' from etcd_server_health_failure, got %q", hv)
 	}
 }

